sites/mosul/configs/openmrs/initializer_config/roles/roles_mosul.csv

```diff
@@ -6,7 +6,7 @@ ec48c936-a9fc-4e32-9a10-06186a7ff19d,Mosul: View patient queues,Able to view pat
 8cac4dde-157c-42af-a9cf-71643af3ec4e,Mosul: Manage appointments,Able to manage patient appointments.,Mosul: General role,Manage Appointments; Appointments: Invite Providers; Manage Appointment Services; Manage Appointment Specialities; Manage Own Appointments; Reset Appointment Status; View Appointment Services; View Appointments
 e9c625fa-f464-4fff-9852-1105fdb1373a,Mosul: Manage Patients,Able to create edit and delete patients and view patient summary.,Mosul: General role,Patient Dashboard - View Patient Summary; Manage Patient Medication; Configure Visits; Patient Overview - View Allergies; Patient Dashboard - View Conditions; Patient Dashboard - View Encounters Section; View Concepts; Manage Immunization Summary; Manage Order Basket; Manage Visit Note; Manage Clinical Forms; Patient Overview - View Programs; Manage Patient Orders; Manage Patient Attachments; View Orders
 959af108-55c9-4a33-a8ef-86a7c9314877,Mosul: Nutrition,Able to view and manage nutrition data.,Mosul: General role; Mosul: Manage Patients,Manage Nutrition,
-4a6105ee-25ae-4880-9882-fc3910c865ab,Mosul: Mental Health,Able to view and manage mental health data.,Mosul: General role; Mosul: Manage Patients,Manage Mental Health,
+4a6105ee-25ae-4880-9882-fc3910c865ab,Mosul: Mental Health,Able to view and manage mental health data.,Mosul: General role; Mosul: Manage Patients,Manage Mental Health; Manage Sensitive Form Fields, 
```

Comment:
This shouldn't belong to the whole Mental Health Role but only the MH doctors and Counselor should be able to edit the field based on the ticket